notebooks/visualization/Cartopy-Intro.ipynb

```diff
@@ -0,0 +1,389 @@
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "fig = plt.figure(figsize=(10, 8))\n",
+    "ax = fig.add_subplot(1, 1, 1, projection=ccrs.LambertConformal())\n",
+    "\n",
+    "ax.add_feature(cfeature.COASTLINE)\n",
+    "ax.add_feature(cfeature.BORDERS, linewidth=2)\n",
+    "ax.add_feature(cfeature.STATES, linestyle='--', edgecolor='black')\n",
+    "\n",
+    "# Set the projection of the data point such that it transforms the point from lon, lat to the projected coordinate system Lambert Conformal\n",
+    "data_projection = ccrs.PlateCarree()\n",
+    "ax.plot(-105, 40, marker='o', color='tab:red', transform=data_projection)\n",
+    "\n",
+    "ax.set_extent([-130, -60, 20, 55])"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "This approach by CartoPy separates the data coordinate system from the coordinate system of the plot. It allows you to take data in any coordinate system (lon/lat, Lambert Conformal) and display it in any map you want. It also allows you to combine data from various coordinate systems seamlessly. This extends to all plot types, not just plot:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Create some synthetic gridded wind data\n",
+    "import numpy as np\n",
+    "from metpy.calc import wind_speed\n",
+    "from metpy.units import units\n",
+    "\n",
+    "# Note that all of these winds have u = 0 -> south wind\n",
+    "v = (np.full((5, 5), 10, dtype=np.float64) + 10 * np.arange(5)) * units.knots\n",
+    "u = np.zeros_like(v) * units.knots\n",
+    "speed = wind_speed(u, v)\n",
+    "\n",
+    "# Create arrays of longitude and latitude\n",
+    "x = np.linspace(-120, -60, 5)\n",
+    "y = np.linspace(30, 55, 5)\n",
+    "\n",
+    "# Plot as normal\n",
+    "fig = plt.figure(figsize=(10, 8))\n",
+    "ax = fig.add_subplot(1, 1, 1, projection=ccrs.LambertConformal())\n",
+    "\n",
+    "ax.add_feature(cfeature.COASTLINE)\n",
+    "ax.add_feature(cfeature.BORDERS)\n",
+    "\n",
+    "# Plot wind barbs--CartoPy handles reprojecting the vectors properly for the\n",
+    "# coordinate system\n",
+    "ax.barbs(x, y, u.m, v.m, transform=ccrs.PlateCarree(), color='tab:blue')\n",
+    "\n",
+    "ax.set_extent([-130, -60, 20, 55])"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "<a href=\"#top\">Top</a>\n",
+    "\n",
+    "---"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Check out these resources"
```

Comment:
I might suggest adding a ###Related Notebooks subsection and referencing the Matplotlib: Basics notebook (https://nbviewer.jupyter.org/github/Unidata/pyaos-ams-2021/blob/master/notebooks/visualization/matplotlib-basics.ipynb) since that's used in this notebook.